### PR TITLE
OP-153 lot information at the wards level fixes

### PR DESCRIPTION
**Pull request by mwithi**
Small fixes related to OP-153.

### CODE REVIEW

File: src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java [hunk 1]
```diff
@@ -958,13 +946,24 @@ private boolean checkAndPrepareMovements() {
 			mov.setWard((Ward) jComboBoxDestination.getSelectedItem());
 			mov.setDate(thisDate);
 			mov.setRefNo(jTextFieldReference.getText());
+			quantities.add(mov.getQuantity());
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java:949]: why is `quantities` an `ArrayList<Integer>` when `getQuantity()` returns an `int`?  Make the array the same type so there is no boxing required.

File: src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java [hunk 2]
```diff
@@ -958,13 +946,24 @@ private boolean checkAndPrepareMovements() {
...
 		return ok;
 	}
 	
+	private void rollBackMovements() {
+		ArrayList<Movement> movements = model.getMovements();
+
+		// Set back changed quantities
+		for (int i = 0; i < movements.size(); i++) {
+			Movement mov = movements.get(i);
+			mov.setQuantity(quantities.get(i));
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java:963]: here is another place that boxing would be required if not changed to `int`.

**Comment by dbmalkovsky** [comment]
Other than the comment; LGTM.

File: src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java:949]: @dbmalkovsky could you please suggest the code modification directly?

Comment (dbmalkovsky) [reply src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java:949]: Nevermind; I was wrong.

**Pull request merged**